app/models/campus_oracle/queries.rb

```diff
@@ -375,6 +417,8 @@ def self.terms
     private
 
     def self.get_course_sections(term_yr, term_cd, department, catalog_id, only_secondary_sections)
+      self.raise_legacy_error if Rails.env != 'test'
+
```

Comment:
Do you think it's worth it to move the environment check for test into CampusOracle::Connection.raise_legacy_error itself?

```ruby
def self.raise_legacy_error
  return if Rails.env.test?
	error_msg = "CampusOracle Query call detected"
	Rails.logger.error error_msg
	raise StandardError, error_msg
end

Reply:
@developish - Good point